pkgs/native_assets_cli/test/model/build_output_test.dart

```diff
@@ -275,6 +178,7 @@ version: 1.0.0''';
       () => HookOutputImpl.fromJsonString('''timestamp: 2022-11-10 13:25:01.000
 assets:
   - name: foo
+    type: native_code
```

Comment:
A test with version 1.0.0 doesn't make sense if we remove it?

Reply:
Removed